06-building-a-program-start-to-finish/writing-tests/output/invertedIndexSpec.js

```diff
@@ -0,0 +1,161 @@
+const books = require('../jasmine/assets/correct.json');
+const wrongdata = require('../jasmine/assets/wrongFormat.json');
+const emptyfile = require('../jasmine/assets/emptyfile.json');
+const smallfile = require('../jasmine/assets/smallcorrectfile.json');
+const uniqueTermsBook = require('../jasmine/assets/uniqueTermsBook.json');
+
+describe('InvertedIndex Class', () => {
+  beforeAll(() => {
+    this.invertedIndex = new InvertedIndex();
+  });
+
+  describe('Constructor', () => {
+    it('can create inverted index instance', () => {
+      expect(typeof this.invertedIndex)
+        .toEqual('object');
+      expect(this.invertedIndex instanceof InvertedIndex)
+        .toBe(true);
+    });
+
+    it('has an indexes object to hold all indexes', () => {
+      expect(typeof this.invertedIndex.tableObj)
+        .toEqual('object');
+    });
+  });
+
+  describe('tokenize', () => {
+    it('checks the tokenize function', () => {
```

Comment:
I'm not sure I understand what this spec is testing for `checks the tokenize function`